### DISCUSSION TITLE
Brand new install.  Unable to use Hifiberry DAC 2 HD

### DISCUSSION THREAD

**Discussion by apastuszak** [rern/rAudio#992] [General]
Wiped my SD Card and did a reinstall of the latest image. Then I updated raudio1 and updated Arch and rebooted.

I enabled I2S, picked Hifiberry DAC 2 HD and rebooted.

I get no sound and my Player preferences look like this:

![image](https://user-images.githubusercontent.com/3401054/186211902-232c30e9-55b7-4b64-bf84-2c89ba9b6a12.png)

This is what's in my config.txt

```
gpu_mem=32
initramfs initramfs-linux.img followkernel
max_usb_current=1
disable_splash=1
disable_overscan=1
dtparam=krnbt=on
dtparam=i2s=on
dtoverlay=hifiberry-dacplushd
```

**Comment by apastuszak** [comment]
Ok, I got it fixed.  Needed to add this line to my config.txt

force_eeprom_read=0

**Reply by pi4user** [reply]
Oh man, you are my hero!!  I have struggled with no sound through  my DAC2HD for days and days and the above script has worked a treat. Many thanks indeed
